suite-common/trading/src/utils.ts

```diff
@@ -427,3 +427,57 @@ export const getTokenSelectableNetworks = (
                 TOKEN_SELECT_SELECTABLE_NETWORKS.includes(n.symbol),
         )
         .map(n => n.symbol);
+
+// reserve defined in networksConfig.ts
+// applies to the native token only
+export const getNetworkReserve = (
+    symbol: NetworkSymbol,
+    contractAddress?: string,
```

Comment:
nit:
if it can be also null, then you dont have to do `contractAddress ?? undefined` on some places and it does not change anything inside this function
```suggestion
    contractAddress?: string | null,
```

Reply:
done